Mention helpdesk tickets for score entry (#184)
srobo/runbook: docs/competition/matches/match-scoring.md

There are two score sheets for every match. It's likely that there may be a discrepancy between the sheets. If that's the case, one match score entry volunteer should speak to the marshals to try to find the correct state.

> If you have any questions, speak to the competition software co-ordinator, or the head judge if it is a judging decision. The head judge should be reachable by radio, the competition software co-ordinator is usually on the balcony.

Reviewer: ```suggestion
If you have any questions, speak to the competition software co-ordinator, or the head judge if it is a judging decision. The competition software co-ordinator should be reachable by radio and the head judge is frequently in the cube.
```
The head judge sadly doesn't have a radio this year

There are two score sheets for every match. It's likely that there may be a discrepancy between the sheets. If that's the case, one match score entry volunteer should speak to the marshals to try to find the correct state.

> If you have any questions, speak to the head match scorer. In SR2023 the head match scorer can be called down from the balcony.

Reviewer: Who is the head match scorer? In SR2022, we didn't have a head match scorer due to the change in how match ops worked. Is this more referring to myself/Alistair now?

Author: Good point, have changed it to the Judge/Competition Software Co-ordinator, based  on the issue.
